cibuildwheel/util.py

```diff
@@ -136,33 +140,37 @@ class Architecture(Enum):
     x86 = 'x86'
     AMD64 = 'AMD64'
 
+    # Allow this to be sorted
+    def __lt__(self, other: "Architecture") -> bool:
+        return self.value < other.value
+
```

Comment:
To nitpick on sómething: I'm not sure I like alphabetic ordering, here. Isn't there a way we can keep the same kind of ordering as usual?

Reply:
(I.e., the same as in `get_python_configurations`/the order they're listed here)

Reply:
What about keeping `allowed_architectures` a list, as it was?

Reply:
It’s treated as a set other places, so this is more natural I think. I could probably keep the same order as defined. Though arches don’t have an order? Order by age?

Reply:
I thought of "order by 'this is the order in which we always list them'", i.e. "x86_64 i686 aarch64 ppc64le s390x" (plus Windows at the end AFAIC).

> It’s treated as a set other places, so this is more natural I think.

It's just a local thing, when you're testing/printing in `build` (and sorting to get a list ;-) ), so yes, you're right, but ... eh, that might just make thing easier?

I guess it's not a blocker, but I find it weird that `aarch64` would suddenly suddenly first when printing a message on Linux.

Reply:
Let me give it a shot.

Reply:
Done, sorts in definition order now.

Reply:
Thanks, that's great! :-)